src/gui/attributeformconfig/qgsattributeformcontaineredit.h

```diff
@@ -19,24 +19,38 @@
 // We don't want to expose this in the public API
 #define SIP_NO_FILE
 
-#include <QWidget>
-
+#include "qgsattributesformmodel.h"
+#include "qgis_gui.h"
 #include "ui_qgsattributeformcontaineredit.h"
 
-#include "qgis_gui.h"
+#include <QWidget>
 
-class QTreeWidgetItem;
 
 /**
- * Widget to edit a container (tab or group box) of a form configuration
+ * Widget to edit a container (tab, group box, or row) of a form configuration.
+ *
  * \since QGIS 3.14
  */
 class GUI_EXPORT QgsAttributeFormContainerEdit : public QWidget, private Ui_QgsAttributeFormContainerEdit
 {
     Q_OBJECT
 
   public:
-    explicit QgsAttributeFormContainerEdit( QTreeWidgetItem *item, QgsVectorLayer *layer, QWidget *parent = nullptr );
+    explicit QgsAttributeFormContainerEdit( const QgsAttributesFormTreeData::DnDTreeNodeData &nodeData, QgsVectorLayer *layer, QWidget *parent = nullptr );
+
+    /**
+     * Sets the \a containerName.
+     *
+     * \since QGIS 3.44
+     */
+    void setTitle( const QString &containerName );
+
+    /**
+     * Sets up the container type comboBox based on the \a containerType and on whether it \a isTopLevelContainer.
```

Comment:
So, beyond the type (comboBox), I feel like when we use \a to refer to parameter within a function description, it should mostly be reserved for instances where it reads like English (i.e. Sets the field \a name). Here it's not really the case, so I'd go for a list of \param

```suggestion
     * Sets up the container combo box based on its type and whether it is located at the top level.
       \param isTopLevelContainer container sitting at the top level
       \param containerType container type
```